packages/frontend-2/components/project/page/settings/general/block/Delete/Delete.vue

```diff
@@ -39,6 +39,9 @@ graphql(`
     workspace {
       slug
     }
+    automations {
```

Comment:
set limit:0, if possible, we sometimes use that to shortcut query execution on the backend